client-js/app/elements/labrad-nodes.ts

```diff
@@ -626,47 +636,43 @@ export class LabradNodes extends polymer.Base {
   }
 
 
-  private updateServerAutostartStatus(serversName: string,
-                                      serverIdx: number): void {
-    const server = this[serversName][serverIdx];
-    server.autostart = false;
-    for (const node of server.nodes) {
-      if (node.exists && node.autostart) {
-        server.autostart = true;
-        return;
+  private updateServerVersion(listName: string, serverIdx: number): void {
+    const server = this[listName][serverIdx];
+    for (let nodeIdx = 0; nodeIdx < server.nodes.length; nodeIdx++) {
+      const node = server.nodes[nodeIdx];
+      if (node.exists) {
+        const status = this.getServerStatus(node.name, server.name);
+        const version = status.outdated
+                      ? `running=${status.runningVersion}, latest=${status.latestVersion}`
+                      : status.version;
+        if (node.version !== version) {
+          this.set(`${listName}.#${serverIdx}.nodes.#${nodeIdx}.version`,
+                   version);
```

Comment:
Might not be worth the effort, but is it possible to mutate the array normally, then call [`notifySplices`](https://www.polymer-project.org/2.0/docs/devguide/model-data#notifysplices) to refresh the whole thing at once?